packages/web-react/src/components/UNSTABLE_Toggle/__tests__/UNSTABLE_Toggle.test.tsx

```diff
@@ -0,0 +1,92 @@
+import { restPropsTest } from '../../../../tests/providerTests/restPropsTest';
+import { stylePropsTest } from '../../../../tests/providerTests/stylePropsTest';
+import UNSTABLE_Toggle from '../UNSTABLE_Toggle';
+
+describe('UNSTABLE_Toggle', () => {
+  classNamePrefixProviderTest(UNSTABLE_Toggle, 'UNSTABLE_Toggle');
+
+  stylePropsTest(UNSTABLE_Toggle);
+
+  restPropsTest(UNSTABLE_Toggle, 'input');
+
+  validationStatePropsTest(UNSTABLE_Toggle, 'UNSTABLE_Toggle--');
+
+  requiredPropsTest(UNSTABLE_Toggle, 'checkbox', 'id', 'example-id');
+
+  it('should have correct className', () => {
+    render(<UNSTABLE_Toggle id="test-toggle" label="Toggle Label" />);
+
+    expect(screen.getByRole('checkbox').parentElement).toHaveClass('UNSTABLE_Toggle');
+  });
+
+  it('should have label classname', () => {
+    render(<UNSTABLE_Toggle id="test-toggle" label="Toggle Label" />);
+
+    expect(screen.getByText('Toggle Label')).toHaveClass('UNSTABLE_Toggle__label');
+    expect(screen.getByText('Toggle Label')).toContainHTML('label');
+  });
+
+  it('should have label with required classname', () => {
+    render(<UNSTABLE_Toggle id="test-toggle" label="Toggle Label" isRequired />);
+
+    expect(screen.getByText('Toggle Label')).toHaveClass('UNSTABLE_Toggle__label');
+    expect(screen.getByText('Toggle Label')).toHaveClass('UNSTABLE_Toggle__label--required');
+  });
+
+  it('should have hidden classname', () => {
+    render(<UNSTABLE_Toggle id="test-toggle" label="Toggle Label" isLabelHidden />);
+
+    const label = screen.getByText('Toggle Label');
+
+    expect(label).toHaveClass('UNSTABLE_Toggle__label');
+    expect(label).toHaveClass('UNSTABLE_Toggle__label--hidden');
+  });
+
+  it('should have input classname', () => {
+    render(<UNSTABLE_Toggle id="test-toggle" label="Toggle Label" />);
+
+    expect(screen.getByRole('checkbox')).toHaveClass('UNSTABLE_Toggle__input');
+  });
+
+  it('should have helper text with correct classname', () => {
+    render(<UNSTABLE_Toggle id="test-toggle" label="Toggle Label" helperText="Helper Text" />);
+
+    const helperText = screen.getByText('Helper Text');
+
+    expect(helperText).toBeInTheDocument();
+    expect(helperText).toHaveClass('UNSTABLE_Toggle__helperText');
+  });
+
+  it('should have correct attribute when checked', () => {
+    render(<UNSTABLE_Toggle id="test-toggle" label="Toggle Label" isChecked />);
+
+    expect(screen.getByRole('checkbox')).toBeChecked();
+  });
+
+  it('should have correct attribute when disabled', () => {
+    render(<UNSTABLE_Toggle id="test-toggle" label="Toggle Label" isDisabled />);
+
+    expect(screen.getByRole('checkbox')).toBeDisabled();
+  });
+
+  it('should have correct classname if fluid', () => {
+    render(<UNSTABLE_Toggle id="test-toggle" label="Toggle Label" isFluid />);
+
+    expect(screen.getByRole('checkbox').parentElement).toHaveClass('UNSTABLE_Toggle UNSTABLE_Toggle--fluid');
```

Comment:
why here you have one expect for two classes and above you have two for two classes? :)

Reply:
Mistake